scripts/extract-release-changelog.py

```diff
@@ -23,8 +30,11 @@ def main():
     )
     args = parser.parse_args()
 
-    tag = get_tag()
-    url = f"https://github.com/lablup/backend.ai/blob/{tag}/CHANGELOG.md"
+    prev_tag, tag = get_prev_tag(), get_tag()
+    url = {
+        "commit_log": f"https://github.com/lablup/backend.ai/compare/{prev_tag}...{tag}",
+        "change_log": f"https://github.com/lablup/backend.ai/blob/{tag}/CHANGELOG.md"
+    }
```

Comment:
just a question.
why did you change `url` into dictionary?
we can just make multiple `..._url` strings.

Reply:
I've just thought that using a dictionary would be convenient to handle urls. (in the case of adding new URLs related to logs).